chart/values.yaml

```diff
@@ -14,6 +14,7 @@ prometheus:
   namespace: "monitoring"                    # K8s namespace where prometheus is installed
   namespace_am: "monitoring"                 # K8s namespace where prometheus-alertmanager is installed
   endpoint: "http://prometheus-server.monitoring.svc.cluster.local:80"   # HTTP Endpoint for Prometheus
+  scrapeInterval: 5s                         # Prometheus scrape interval. Value is a time duration expressed as a sequence of decimal numbers followed by unit suffixes such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
```

Comment:
should this be a string here? e.g., `"5s"`?

Reply:
No, it worked for me when running it. Do you still want to change it, so it matches the formatting used in the variables above?

Reply:
If it works, we can leave it as it is.